src/charm.py

```diff
@@ -56,12 +59,24 @@ def __init__(self, *args):
         self.grafana_source_provider = GrafanaSourceProvider(
             charm=self,
             refresh_event=self.on.loki_pebble_ready,
-            source_type="loki",
+            source_type=PEER,
```

Comment:
This seems a find/replace mistake. This is the datasource type. It just happens now to have the same name as the peer relation.